misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/central/CentralClientWrapper.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.semver.checker.central;
+
+import io.ballerina.projects.JvmTarget;
+import io.ballerina.projects.SemanticVersion;
+import io.ballerina.projects.Settings;
+import io.ballerina.projects.util.ProjectConstants;
+import io.ballerina.projects.util.ProjectUtils;
+import io.ballerina.semver.checker.exception.SemverToolException;
+import org.ballerinalang.central.client.CentralAPIClient;
+import org.ballerinalang.central.client.exceptions.CentralClientException;
+import org.ballerinalang.central.client.exceptions.PackageAlreadyExistsException;
+import org.ballerinalang.toml.exceptions.SettingsTomlException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.ballerinalang.util.RepoUtils;
+
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+
+import static io.ballerina.projects.util.ProjectUtils.getAccessTokenOfCLI;
+import static io.ballerina.projects.util.ProjectUtils.initializeProxy;
+
+/**
+ * Ballerina central client wrapper implementation.
+ *
+ * @since 2201.2.0
+ */
+public class CentralClientWrapper {
+
+    private final PrintStream outStream;
```

Comment:
Do we have usages of `outStream`, Or added for future usages?

Reply:
Currently we don't have any usages. I've added it for the future references :)